.github/workflows/ci.yml

```diff
@@ -207,6 +207,26 @@ jobs:
         env:
           DYLD_ROOT_PATH: "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot"
 
+  sys-crate-tests:
+    name: sys crate tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest, macos-12 ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - uses: actions-rs/toolchain@v1.0.7
+        id: toolchain
+        with:
+          toolchain: stable
+          override: true
+      - name: Run cargo test
+        working-directory: ./aws-lc-fips-sys-testing
```

Comment:
Do we also need to do the `aws-lc-sys-testing` crate? I wasn't sure if the CI automatically picks up new testing crates?

Reply:
From my quick look-through I believe we do. Feel free to copy this over, or I can update this PR once yours is merged in so that it includes the new testing crate you added.

Reply:
I saw that it was being picked up in the [coverage build](https://github.com/aws/aws-lc-rs/actions/runs/6881835036/job/18719050888?pr=277#step:5:753) and figured that was good enough, although I guess it's missing the macOS build. But since we'd conflict if I pulled this in, it's probably better to just add it as part of the matrix in yours.

Reply:
Ah I think code coverage was the only one slurping it up since I believe we execute it from the root of the repository. Probably worth having an explicit invocation on it though, so will update this once we get your change merged in.